JS/Lesson22/goksel/style.css

```diff
@@ -0,0 +1,105 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    font-family: sans-serif;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background-color: #000
+}
+
+.content-header {
+    text-align: center;
+    font-size: 24px;
+    padding: 12px;
+    margin: 5px;
+    font-weight: 600;
+}
+
+.accordion {
+    max-width: 600px;
+    background-color: #fff;
+    border-radius: 5px;
+    padding: 10px;
+}
+
+.accordion-container{ 
+    border: 1px solid #000;
+    border-radius: 5px;
+    margin: 5px;
+}
+
+
+
+.click-btn {
+display: flex;
+align-items: center;
+justify-content: space-between;
+padding: 10px 15px;
+font-size: 24px;
+font-weight: 500;
+cursor: pointer;
+color: #fff;
+background-color: 	#4e6fc2;
+transition: .5s;
+}
+
+.click-btn::after {
+    content: "+";
+}
+
+.content {
+    padding: 0px 15px;
+    line-height: 1.5;
+    font-size: 24;
+    height: 0;
+    overflow: hidden;    
+}
+
+/* these 2 class will work after javascript click function worked */
+
+.click-btn.active::after{
+    content: "-";
+    font-size: 34px;
+
+}
+
+.content.active {
+    overflow: auto;
+    height: auto;
+}
+
+
+/*added some extra class and id for the design links inside of the content*/
+
+.link-btn {
+    text-decoration: none;
+    display: block;
+    text-align: center;
+}
+
+.link-btn:hover {
+    font-size: 48px;
+    transition: .5s;
+}
+
+#mdn {
+    color: #0b45b1;
+}
+
+#mdn-btn {
+    color: #1f048a;
+}
+
+#code {
+    border: 2px solid #000
+}
+
+/* Ä± could not find how to change underline of c to black  */
+#c {
+    text-decoration: underline;  
```

Comment:
You cannot do that using underline as the line under the letter is considered part of the word so it will always be in the color of the text. 
You could instead use a ::after pseudo-selector:

```
#c {
    position: relative;
    text-decoration: none;
}

#c::after {
    content: '';
    position: absolute;
    left: 1px;
    bottom: -2px; /* Adjust this value as needed */
    width: calc(100% - 1px);
    height: 2px; /* Adjust thickness of the underline */
    background-color: black;
}
```